dist/countdown/countdown.js

```diff
@@ -0,0 +1,89 @@
+const countDownApp = Vue.createApp({
+    template:`<count-down :timerDuration="timerDuration" @notify-to-app-for-seconds="updateTimerDuration"> </count-down>`,
+    data() {
+        return{
+            timerDuration:0
+        }
+    },
+    watch:{
+        timerDuration() {
+            if(this.timerDuration === 0){
+                onCountDownTimerComplete();
+            }
+        }
+    },
+    methods:{
+        updateTimerDuration(value){
+            this.timerDuration=value;
+        }
+    }
+});
+
+countDownApp.component('count-down',{
+    template:`
+        <div>
+            <span class="h-l">{{hl}}</span>
+            <span class="h-r">{{hr}}</span>
+        </div>
+        <div>
+            <span class="devider">:</span>
+        </div>
+        <div>
+            <span class="m-l">{{ml}}</span>
+            <span class="m-r">{{mr}}</span>
+        </div><div>
+            <span class="devider">:</span>
+        </div>
+        <div>
+            <span class="s-l">{{sl}}</span>
+            <span class="s-r">{{sr}}</span>
+        </div>
+    `,
+    props:{
+        timerDuration:Number
+    },
+    watch:{
+        timerDuration() {
+            if(this.timerDuration>0){
+                this.totalSeconds = this.timerDuration;
+                this.startTimer();
+            }
+        }
+    },
+    data() {
+        return {
+            totalSeconds:0,
+            hr:"",
+            hl:"",
+            mr:"",
+            ml:"",
+            sr:"",
+            sl:""
+        }
+    },
+    emits:['notify-to-app-for-seconds'],
+    methods:{
+        startTimer(){
+            const thisTimer = this;
+            let intervalTimer = setInterval(function(){
+                let seconds = parseInt(thisTimer.totalSeconds, 10);
+                let hours   = Math.floor(seconds / 3600); // get hours
+                let minutes = Math.floor((seconds - (hours * 3600)) / 60); // get minutes
+                seconds = seconds - (hours * 3600) - (minutes * 60); //  get seconds
```

Comment:
```suggestion
                let seconds = parseInt(thisTimer.totalSeconds, 10);
                let hours   = Math.floor(seconds / 3600); // get hours
                let minutes = Math.floor((seconds - (hours * 3600)) / 60); // get minutes
                seconds = seconds - (hours * 3600) - (minutes * 60); //  get seconds
```
let hours   = Math.floor(totalSeconds / 3600);
let minutes = Math.floor(totalSeconds /60) % 60;
let seconds = totalSeconds % 60;